spec/overcommit/hook/pre_commit/rake_target_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe Overcommit::Hook::PreCommit::RakeTarget do
+  let(:config)  { Overcommit::ConfigurationLoader.default_configuration }
+  let(:context) { double('context') }
+  subject { described_class.new(config, context) }
+
+  # context 'without targets parameters' do
+  #   let(:result) { double('result') }
+  #   it 'raises' do
+  #     expect { subject }.to raise_error(RuntimeError)
```

Comment:
If you want to test the error case, you can check the content of the error message rather than the type of error using RSpec's regex matcher, e.g.

```ruby
expect { subject }.to raise_error(/some message/i)
```